tests/sysauth/sssd.pm

```diff
@@ -39,7 +40,16 @@ sub run {
     # The test scenarios are now ready to run
     my @scenario_failures;
 
-    foreach my $scenario (qw(local ldap ldap-no-auth ldap-nested-groups krb)) {
+    my @scenario_list;
+    push @scenario_list, 'local' if (!is_tumbleweed);    # sssd 2.0+ removed support of 'local'
```

Comment:
Can we do more generic check, e.g. of the sssd version here? By doing so we won't have to touch this code again if package will be accepted for other distributions. WDYT?

Reply:
Yes, it could be done with ```rpm -q sssd --qf '%{VERSION}'```, and compared with ```perl::version```.
Let's give it a try.